ddcommon-ffi/src/string.rs

```diff
@@ -72,3 +73,45 @@ pub unsafe extern "C" fn ddog_StringWrapper_message(s: Option<&StringWrapper>) -
         Some(s) => CharSlice::from(s.as_ref()),
     }
 }
+
+pub trait ToHexStr {
```

Comment:
I'm not sure this is the file I'd expect to find this in.

Reply:
Where would you put it ?
I thought about this file since it was about string 🤔

Reply:
Maybe in utils? This file is more about the `CharSlice` FFI implementation of string.